### ISSUE TITLE
Not working in private repos (404)

### ISSUE THREAD

**Issue by nmaxcom** [harshjv/github-repo-size#34]
Even though I generated the token and added to the extension I get this in the console:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
inject.js:109 Error: GitHub returned a bad status: 404
    at checkStatus (inject.js:79)
```

When I try that 404 fetch myself I see this:

![image](https://user-images.githubusercontent.com/8936090/51865970-b20d1d00-2315-11e9-9701-830a6e836178.png)

**Comment by harshjv** [comment]
Strange! It is working for me on private repos. Do you have permission to view that private repo?

**Comment by nmaxcom** [comment]
Yes, as you can tell from the URL and my username, it is my private repo 😂

**Comment by harshjv** [comment]
Haha. Ya, missed it the first time.

I am unable to reproduce the results. It is working correctly on my private repos.

Did you try the latest version of this extension? It's 0.5.0.

**Comment by nmaxcom** [comment]
I'm sorry, I'm way over my head in work. Otherwise, I'd dig a bit and find the bug. Could you link me the new version?
